src/themes/default/css/border.scss

```diff
@@ -0,0 +1,55 @@
+// IMPORTANT: Add new css rules ON DEMAND only when you need them! 
+// Try to use multiples of 5 when using pixels
+
+// Pixel sizes
+
+$border-px: 1;
+
+// Display
+
+$border-display: none;
+
+// Colors
+
+$border-colors: (
```

Comment:
Colors should be imported from colors.scss or you can move all global variables to vars.scss